addons/bestja_estimation_reports/models.py

```diff
@@ -0,0 +1,194 @@
+
+    estimation_reports = fields.One2many('bestja.estimation_report', inverse_name='project')
+    enable_estimation_reports = fields.Boolean(string="Raporty szacunkowe do zbiórki żywności")
+
+    use_estimation_reports = fields.Boolean(
+        compute='_compute_use_estimation_reports',
+        compute_sudo=True,
+        search='search_use_estimation_reports',
+    )
+
+    @api.one
+    @api.depends(
+        'enable_estimation_reports',
+        'parent.enable_estimation_reports',
+        'parent.parent.enable_estimation_reports'
+    )
+    def _compute_use_estimation_reports(self):
+        self.use_estimation_reports = (
+            self.enable_estimation_reports or
+            self.parent.enable_estimation_reports or
+            self.parent.parent.enable_estimation_reports
+        )
+
+    def search_use_estimation_reports(self, operator, value):
+        return [
+            '|',  # noqa
+                ('enable_estimation_reports', operator, value),
+            '|',
+                ('parent.enable_estimation_reports', operator, value),
+                ('parent.parent.enable_estimation_reports', operator, value),
+        ]
+
+
+class EstimationReportEntry(models.Model):
+    _name = 'bestja.estimation_report_entry'
+
+    estimation_report = fields.Many2one('bestja.estimation_report', ondelete='cascade', required=True)
+    day_in_store = fields.Many2one('bestja_stores.day', required=True, string="dzień")
+    day = fields.Date(
+        related='day_in_store.date',
+        store=True,
+    )
+    tonnage = fields.Float(required=True, string="Liczba kg")
+    store = fields.Many2one(
+        required=True,
+        related='day_in_store.store',
+        store=True,
+    )
+    store_project = fields.Many2one(
+        'bestja.project',
+        required=True,
+        related='estimation_report.project'
+    )
+    store_name = fields.Char(
+        string="Nazwa sklepu",
+        related='day_in_store.store.store.display_name',
+        store=True,
+    )
+    store_address = fields.Char(
+        required=True,
+        related='store.store.address',
+    )
+    store_city = fields.Char(required=True, related='store.store.city')
+    chain = fields.Many2one(
+        related='store.store.chain',
+        store=True,
+    )
+    chain_name = fields.Char(
+        related='chain.name',
+        store=True,
+    )
+    organization = fields.Many2one(
+        'organization',
+        string="Organizacja",
+        related='estimation_report.organization',
+        store=True,
+    )
+
+
+class EstimationReport(models.Model):
+    _name = 'bestja.estimation_report'
+    _inherit = [
+        'protected_fields.mixin',
+        'ir.needaction_mixin',
+    ]
+    _protected_fields = ['state']
+    STATES = [
+        ('sent', 'dodany do zestawienia'),
+        ('draft', 'szkic'),
+    ]
+    project = fields.Many2one(
+        'bestja.project',
+        required=True,
+        string="projekt:",
```

Comment:
Also note that those labels can be used in many different contexts, so I don't think you should put colon here.